app/javascript/components/organizations-dropdown.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <clickable-dropdown
+    :body-title="dropdownTitle"
+    :no-items-message="noOrganizationsMessage"
+    :items="organizationsWithName"
+    :default-index="defaultOrganizationIndex"
+    @item-clicked="onItemClicked"
+  >
+  </clickable-dropdown>
+</template>
+
+<script>
+import ClickableDropdown from './clickable-dropdown';
+import { PROCESS_NEW_ORGANIZATION } from '../store/action-types';
+import FroggoLocalStorage from '../helpers/local-storage';
+
+export default {
+  props: {
+    organizations: Array,
+    githubLogin: String,
+    teams: Array,
+  },
+  data() {
+    const namedOrganizations = this.organizations;
+    namedOrganizations.forEach((organization) =>
+      (organization.name = organization.login));
+
+    return {
+      dropdownTitle: this.$t('message.profile.organizationsDropdownTitle'),
+      noOrganizationsMessage: this.$t('message.profile.noOrganizations'),
+      organizationsWithName: namedOrganizations,
+    };
+  },
+  computed: {
+    defaultOrganizationIndex() {
+      const index = this.getDefaultOrganizationIndex();
+      const selectedOrganization = this.organizations[index];
+      if (selectedOrganization) {
+        this.onOrganizationSelected(selectedOrganization);
```

Comment:
`defaultOrganizationIndex` no debería tener side-effects: querer _computar_ el `default organization index` no debería implicar _avisar_ que se seleccionó una organización (`this.onOrganizationSelected(..)`)